cluster/ubuntu/util.sh

```diff
@@ -188,8 +188,8 @@ function create-etcd-opts() {
   cat <<EOF > ~/kube/default/etcd
 ETCD_OPTS="\
  -name infra\
- -listen-client-urls http://0.0.0.0:4001\
- -advertise-client-urls http://127.0.0.1:4001"
+ -listen-client-urls http://127.0.0.1:4001,http://${1}:4001\
```

Comment:
'cuz i can barely understand the relationship between these two flags(from doc).Shame on me:-/
If you could give more details on this, it would be highly appreciated. 
` -listen-client-urls http://127.0.0.1:4001,http://${1}:4001 -advertise-client-urls http://${1}:4001` would work.